checker/check.go

```diff
@@ -49,28 +49,19 @@ func newCheckData(lastCheck *moira.CheckData, checkTimeStamp int64) moira.CheckD
 	newCheckData.Timestamp = checkTimeStamp
 	newCheckData.Message = ""
 	return newCheckData
-	}
-
-func newMetricState(oldMetricState moira.MetricState, newState moira.State, newTimestamp int64, newValue *float64) *moira.MetricState {
-	newMetricState := oldMetricState
+}
 
+func newMetricState(metricState moira.MetricState, newState moira.State, newTimestamp int64, newValue *float64) *moira.MetricState {
```

Comment:
What is the purpose of returning pointer? As we receive metric state by value, probably it woud be more consistent to return metric state by value.